src/modules/esl-trigger/core/esl-trigger.ts

```diff
@@ -83,6 +88,15 @@ export class ESLTrigger extends ESLBaseElement {
     return this.a11yTarget ? this.querySelector(this.a11yTarget) : this;
   }
 
+  /** Value to setup aria-label */
+  public get a11yLabel(): string | null {
+    if (this.a11yLabelActive || this.a11yLabelInactive) {
+      const label = this.$target?.open ? this.a11yLabelActive : this.a11yLabelInactive;
+      return label ?? '';
+    }
+    return null;
```

Comment:
```suggestion
   if (!this.$target) return null;
   return (this.$target.open ? this.a11yLabelActive : this.a11yLabelInactive) || null;
```
In term of suggestion below